### PR TITLE
feat: New delegate profile: @bpresles

### PR DESCRIPTION
**Pull request by bpresles**

### CODE REVIEW

File: docs/delegates/8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- **[since 2019]** Active member of the Blockchain & Société association - https://blockchainsociete.org/
+
+- **[since 2019]** Active member of the Ğ1 crypto-currency. Contributing developer of the Cesium wallet application - https://git.duniter.org/clients/cesium-grp/cesium
+
+- **[since 2021]** Co-founder of the FideliĞ1 project. Lead developer on this project
+
+    https://git.duniter.org/bpresles/fidelig1card
+
+    https://git.duniter.org/bpresles/fidelig1app
```
Comment (dlecan) [inline docs/delegates/8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e/README.md:39]: These links aren't browseable. Is this intended?

Comment (bpresles) [reply docs/delegates/8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e/README.md:39]: I've no issue to browse them. They are public and accessible to anyone.

Comment (bpresles) [reply docs/delegates/8b000cc3de75b75d2d3e845770dbb13a5d3827d730098f3eb8d921f23b694c2e/README.md]: I fixed the links not being clickable.

**Review by dlecan — APPROVED** [review]

**Comment by dlecan** [comment]
@bpresles content is ok! 
One last thing: for security reasons, all your commits must be signed. 
See https://docs.github.com/articles/about-gpg/

**Comment by bpresles** [comment]
> @bpresles content is ok!
> One last thing: for security reasons, all your commits must be signed.
> See https://docs.github.com/articles/about-gpg/

The commits have been signed.

**Pull request merged**